Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -4413,7 +4413,7 @@ final class WorkspaceTests: XCTestCase {
             DiagnosticsEngineTester(diagnostics) { result in
                 result.check(diagnostic: .contains("artifact of binary target 'A1' failed download: invalid status code 500"), behavior: .error)
                 result.check(diagnostic: .contains("artifact of binary target 'A2' failed extraction: dummy error"), behavior: .error)
-                result.check(diagnostic: .contains("downloaded artifact of binary target 'A3' has an invalid checksum"), behavior: .error)
+                result.check(diagnostic: .contains("checksum of downloaded artifact of binary target 'A3' (6d75736b6365686320746e65726566666964203d2073746e65746e6f6320746e65726566666964) does not match checksum specified by the manifest (a3)"), behavior: .error)
```

Comment:
In this case, is it just coincidence (or for testing purposes) that the checksum is one byte and is the lowercase version of the package name?

Reply:
Ah, right, I see now that they're all like that for test purposes.  That's not the actual checksum, it's the alleged one.  Looks good then.